### DISCUSSION TITLE
Environment / Terrain System & Editor

### DISCUSSION THREAD

**Discussion by AMZN-Liv** [o3de/rfcs#28] [Ideas]
**Is your feature request related to a problem? Please describe**.
 Terrain systems enable users to create landscapes for a game level and make use of environmental effects and systems such as wind. For the initial launch of O3DE, it is possible to make environments for game levels using meshes created in an external program, but it does not ship with a native height-field-based terrain system.  Using external tools and systems will require users to convert meshes, textures, and materials from those external tools into formats usable by O3DE.

**Describe the solution you'd like**
A fully-featured height-field-based terrain editor with a brush system, the ability to raise and lower the terrain height, topography manipulation capabilities, the ability to add bodies of water and roads, vegetation painting, and object snapping-to-terrain.

**Describe alternatives you've considered**
 Environmental authoring tools to work with externally-created meshes instead of a native tool.

**Additional context**
 Including a screenshot of the terrain editor from Lumberyard and links to the documentation for more details about possible feature capabilities. https://docs.aws.amazon.com/lumberyard/latest/userguide/terrain-intro.html

**Comment by magenta404** [comment]
Could this system support non-destructive workflows using layers and/or prefabs? An example is the [UE Landscape system](https://docs.unrealengine.com/4.26/en-US/BuildingWorlds/Landscape/Layers/).

**Comment by ghost** [comment]
It would be nice if there was a way to import (as part of the O3DE terrain tool) QuadSpinner's Gaea landscaping meshes.

**Comment by AMZN-Liv** [comment]
Going to resolve this discussion in favor of the main discussion thread in o3de/o3de: https://github.com/o3de/o3de/discussions/1974 and the proposed RFC: https://github.com/o3de/sig-content/issues/4
